store/tikv/2pc.go

```diff
@@ -381,6 +390,18 @@ func (c *twoPhaseCommitter) prewriteSingleBatch(bo *Backoffer, batch batchKeys)
 		}
 		var locks []*Lock
 		for _, keyErr := range keyErrs {
+			// Check already exists error
+			if alreadyExist := keyErr.GetAlreadyExist(); alreadyExist != nil {
+				key := alreadyExist.GetKey()
+				err := c.txn.us.LookupConditionErr(key)
+				if err == nil {
+					panic(fmt.Sprintf("con:%d, precondition error for key:%s should not be nil", c.connID, key))
+				}
+				log.Errorf("con:%d key: %s already exists", c.connID, key)
```

Comment:
Why log error here? `key already exists` is a normal business logic.

Reply:
I think it is better to log duplicated entry error in TiDB.

Reply:
I agree with tiancaiamao, if we log the duplicated error, there will be too many useless logs.